languages/python/sqlalchemy-oso/sqlalchemy_oso/roles.py

```diff
@@ -0,0 +1,398 @@
+from typing import Any, List
+
+from sqlalchemy.types import Integer, String
+from sqlalchemy.schema import Table, Column, ForeignKey
+from sqlalchemy.ext.declarative import declared_attr
+from sqlalchemy.orm import relationship, backref
+from sqlalchemy import inspect
+
+
+ROLE_CLASSES: List[Any] = []
+
+
+def enable_roles(oso):
+    """Enable the SQLAlchemy Role-Based Access Control base policy. This method activates the following polar rules:
```

Comment:
Ensure that these doc strings get picked up in the auto-generated API documentation.